.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+<!--
+Thank you for contributing to Lychee! We only accept PR to the master branch.
+
+In addition, please describe the benefit to end users; the reasons it does not break any existing features, etc.
+
+If you're pushing a Feature:
+- Title it: "This new feature"
+- Describe what the new feature enables
+- Ensure it doesn't break any feature.
+
+If you're pushing a Fix:
+- Title it: "Fixes the bug name"
+- If it is a fix an an existing issue start the description with `fixes #xxxx` where xxxx is the issue number.
+- Describe how it fixes in a few words.
+- Ensure it doesn't break any feature.
+
+All Pull Requests run with extensive tests for stable and latest versions of PHP. 
+Ensure your tests pass or your PR may be taken down.
+
+Don't worry if your code styling isn't perfect! php-cs-fixer will automatically create a pull request with any style fixes. This allows us to focus on the content of the contribution and not the code style.
```

Comment:
Shouldn't we at least mention `make formatting` and `make phpstan` here? They run just fine locally without any undue burdens (such as the empty DB requirement of `make test`). Sure, it's nice that we have the online fallback for `php-cs-fixer` but I would still stress that it's better to run them locally first before opening a PR...

Reply:
Better ?

Reply:
Yep...